src/components/accessories/patientNewBill/itemsTable/BillItemsTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import Table from "../../table/Table";
+import { header, mockData, order } from "./consts";
+import { currencyFormat } from "../../../../libraries/formatUtils/currencyFormatting";
+import { useTranslation } from "react-i18next";
+
+interface IOwnProps {
+  handleEdit: (row: any) => void;
+  handleDelete: (row: any) => void;
+}
+
+const BillItemsTable: FC<IOwnProps> = ({ handleEdit, handleDelete }) => {
+  const { t } = useTranslation();
+  const label = {
+    type: t("bill.type"),
+    description: t("bill.description"),
```

Comment:
I think that we should rename the label in "Type" and "Description" (now it's Bill.Type and Bill.Description).

Reply:
OK. we translated them.